Source/VisualRecognitionV3/VisualRecognition+UIImage.swift

```diff
@@ -172,8 +172,8 @@ extension VisualRecognition {
     private func saveToDisk(image: UIImage) throws -> URL {
         let filename = UUID().uuidString + ".jpg"
         let directory = NSURL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
-        guard let file = directory.appendingPathComponent(filename) else { throw RestError.encodingError }
-        guard let data = UIImageJPEGRepresentation(image, 0.75) else { throw RestError.encodingError }
+        guard let file = directory.appendingPathComponent(filename) else { throw RestError.encoding(path: filename) }
+        guard let data = UIImageJPEGRepresentation(image, 0.75) else { throw RestError.serialization(values: "classify image") }
```

Comment:
I noticed a few instances where `RestError.encodingError` was being used in places where `RestError.serializationError` was more appropriate. With the way I have it implemented now, `RestError.encoding` only refers to percent-encoding of URLs. I'm thinking it should be renamed to reflect that, like `RestError.urlEncoding`.

Reply:
I like the more specific name.